storm/datadog_checks/storm/storm.py

```diff
@@ -329,9 +333,10 @@ def get_topology_metrics(self, topology_id, interval=60, storm_version=None):
             endpoint = "/api/v1/topology/{}"
 
         params = {'window': interval}
-        return self.get_request_json(endpoint.format(topology_id),
+        r = self.get_request_json(endpoint.format(topology_id),
```

Comment:
not sure why this was added

Reply:
Accidentally left in from previous iterations :) Incoming commit to remove!